pkg/ccl/backupccl/backup_processor.go

```diff
@@ -514,34 +506,34 @@ func runBackupProcessor(
 						FileSummaries: make([]roachpb.RowCount, 0),
 					}
 
-					if len(res.Files) > 1 {
+					if len(KVResponse.Files) > 1 {
 						log.Warning(ctx, "unexpected multi-file response using header.TargetBytes = 1")
 					}
 
-					for i, file := range res.Files {
-						f := BackupManifest_File{
+					for i, file := range KVResponse.Files {
+						partialFile := BackupManifest_File{
 							Span:        file.Span,
-							Path:        file.Path,
```

Comment:
IMO I'd make a separate commit here that deletes the `Path` field from ExportResponse_File (and reserves it) just to prove it was never being set, which makes it obvious that deleting this line is safe/correct (which I think it is)

Reply:
Sure, reverted for now.